src/frontend/index.story.md

```diff
@@ -518,6 +518,29 @@ example:
 }
 ```
 
+## region
+
```

Comment:
too much info. we do not need to explain in this catalog how the data is stored, detail widget behaviour or things common to all widgets (RTL, readonly)
we do need to explain what all the configuration options are or if there aren't any we need to say so explicitly.

try and match the other fields

These fuller feature descriptions can go into the docs section of src/frontend/fields/region-field.story.vue